metadata/RG351P/themes.cfg

```diff
@@ -1,26 +1,26 @@
-351ELEC-Alekfull https://github.com/szalik-rg351/es-theme-351elec-alekfull
-351ELEC-Comicbook https://github.com/szalik-rg351/es-theme-351elec-comicbook
-351ELEC-CRAZYBOY https://github.com/szalik-rg351/es-theme-351elec-crazyboy
-351ELEC-Carbon https://github.com/szalik-rg351/es-theme-351elec-carbon
-351ELEC-Crystal https://github.com/szalik-rg351/es-theme-351elec-crystal
-351ELEC-Switch https://github.com/szalik-rg351/es-theme-351elec-switch
-351ELEC-Pixel https://github.com/szalik-rg351/es-theme-351elec-pixel
-351ELEC-Chicuelo https://github.com/szalik-rg351/es-theme-351elec-chicueloap
-351ELEC-GBZ https://github.com/szalik-rg351/es-theme-351elec-gbz
-351ELEC-Fundamental https://github.com/szalik-rg351/es-theme-351elec-fundamental
-351ELEC-CosmicRise https://github.com/szalik-rg351/es-theme-351elec-cosmicrise
-351ELEC-SuperSweet https://github.com/szalik-rg351/es-theme-351elec-supersweet
-351ELEC-Minimal https://github.com/szalik-rg351/es-theme-351elec-minimal
-351ELEC-EpicCody https://github.com/szalik-rg351/es-theme-351elec-epic-cody
-351ELEC-Roleta https://github.com/szalik-rg351/es-theme-351elec-roleta
-351ELEC-Supreme https://github.com/szalik-rg351/es-theme-351elec-supreme
-351ELEC-SpinMaster https://github.com/szalik-rg351/es-theme-351elec-spin-master
-Handheld-Simple https://github.com/Rican7/es-theme-handheld-simple
+Horizontal https://github.com/szalik-rg351/Horizontal-Szalik
+EpicLikeSzalik https://github.com/szalik-rg351/Epic-Szalik
+Carbon https://github.com/szalik-rg351/es-theme-351elec-carbon
+351Noir https://github.com/szalik-rg351/es-theme-351elec-351noir
+Alekfull https://github.com/szalik-rg351/es-theme-351elec-alekfull
+Comicbook https://github.com/szalik-rg351/es-theme-351elec-comicbook
 Alt-Book https://github.com/bebeidon/es-theme-alt-book-3-2
-351ELEC-EpicBlancheNoir https://github.com/kefren68/es-theme-351ELEC-epicblanchenoir
+CRAZYBOY https://github.com/szalik-rg351/es-theme-351elec-crazyboy
+EpicBlancheNoir https://github.com/kefren68/es-theme-351ELEC-epicblanchenoir
+Crystal https://github.com/szalik-rg351/es-theme-351elec-crystal
+Switch https://github.com/szalik-rg351/es-theme-351elec-switch
+Pixel https://github.com/szalik-rg351/es-theme-351elec-pixel
+Chicuelo https://github.com/szalik-rg351/es-theme-351elec-chicueloap
+GBZ https://github.com/szalik-rg351/es-theme-351elec-gbz
+Fundamental https://github.com/szalik-rg351/es-theme-351elec-fundamental
+CosmicRise https://github.com/szalik-rg351/es-theme-351elec-cosmicrise
+SuperSweet https://github.com/szalik-rg351/es-theme-351elec-supersweet
+Minimal https://github.com/szalik-rg351/es-theme-351elec-minimal
+EpicCody https://github.com/szalik-rg351/es-theme-351elec-epic-cody
+Roleta https://github.com/szalik-rg351/es-theme-351elec-roleta
+Supreme https://github.com/szalik-rg351/es-theme-351elec-supreme
+SpinMaster https://github.com/szalik-rg351/es-theme-351elec-spin-master
+Handheld-Simple-NOT-SUPPORTED https://github.com/Rican7/es-theme-handheld-simple

```

Comment:
Please change this to just Handheld-Simple.   The theme works well for a set of core systems was one of the first ones that we made for 351ELEC.   Not supported implies that the theme doesn't work at all.